model/simulationtests/vrf_fluid_temperature_control.rb

```diff
@@ -0,0 +1,223 @@
+  coolingCoil.autosizeRatedSensibleHeatRatio
+  coolingCoil.setIndoorUnitReferenceSuperheating(5.0)
+
+  indoorUnitEvaporatingTemperatureFunctionofSuperheatingCurve = OpenStudio::Model::CurveQuadratic.new(model)
+  indoorUnitEvaporatingTemperatureFunctionofSuperheatingCurve.setCoefficient1Constant(0)
+  indoorUnitEvaporatingTemperatureFunctionofSuperheatingCurve.setCoefficient2x(0.843)
+  indoorUnitEvaporatingTemperatureFunctionofSuperheatingCurve.setCoefficient3xPOW2(0)
+  indoorUnitEvaporatingTemperatureFunctionofSuperheatingCurve.setMinimumValueofx(0)
+  indoorUnitEvaporatingTemperatureFunctionofSuperheatingCurve.setMaximumValueofx(15)
+  coolingCoil.setIndoorUnitEvaporatingTemperatureFunctionofSuperheatingCurve(indoorUnitEvaporatingTemperatureFunctionofSuperheatingCurve)
+
+  heatingCoil = OpenStudio::Model::CoilHeatingDXVariableRefrigerantFlowFluidTemperatureControl.new(model)
+  heatingCoil.setAvailabilitySchedule(alwaysOn)
+  heatingCoil.autosizeRatedTotalHeatingCapacity
+  heatingCoil.setIndoorUnitReferenceSubcooling(5.0)
+  indoorUnitCondensingTemperatureFunctionofSubcoolingCurve = OpenStudio::Model::CurveQuadratic.new(model)
+  indoorUnitCondensingTemperatureFunctionofSubcoolingCurve.setCoefficient1Constant(-1.85)
+  indoorUnitCondensingTemperatureFunctionofSubcoolingCurve.setCoefficient2x(0.411)
+  indoorUnitCondensingTemperatureFunctionofSubcoolingCurve.setCoefficient3xPOW2(0.0196)
+  indoorUnitCondensingTemperatureFunctionofSubcoolingCurve.setMinimumValueofx(0)
+  indoorUnitCondensingTemperatureFunctionofSubcoolingCurve.setMaximumValueofx(20)
+  heatingCoil.setIndoorUnitCondensingTemperatureFunctionofSubcoolingCurve(indoorUnitCondensingTemperatureFunctionofSubcoolingCurve)
+
+  fan = OpenStudio::Model::FanVariableVolume.new(model)
+  fan.setAvailabilitySchedule(alwaysOn)
+  fan.setFanTotalEfficiency(0.6045)
+  fan.setPressureRise(1017.592)
+  fan.autosizeMaximumFlowRate
+  fan.setFanPowerMinimumFlowRateInputMethod('FixedFlowRate')
+  fan.setFanPowerMinimumFlowFraction(0.0)
+  fan.setFanPowerMinimumAirFlowRate(0.0)
+  fan.setMotorEfficiency(0.93)
+  fan.setMotorInAirstreamFraction(1.0)
+  fan.setFanPowerCoefficient1(0.0407598940)
+  fan.setFanPowerCoefficient2(0.08804497)
+  fan.setFanPowerCoefficient3(-0.072926120)
+  fan.setFanPowerCoefficient4(0.9437398230)
+  fan.setFanPowerCoefficient5(0.0)
+
+  vrf_terminal = OpenStudio::Model::ZoneHVACTerminalUnitVariableRefrigerantFlow.new(model, coolingCoil, heatingCoil, fan)
+
+  vrf_terminal.addToThermalZone(z)
+  vrf_fluid_temperature_control.addTerminal(vrf_terminal)
+end
+
+# add thermostats
+model.add_thermostats({ 'heating_setpoint' => 24,
+                        'cooling_setpoint' => 28 })
+
+# assign constructions from a local library to the walls/windows/etc. in the model
+model.set_constructions
+
+# set whole building space type; simplified 90.1-2004 Large Office Whole Building
+model.set_space_type
+
+# add design days to the model (Chicago)
+model.add_design_days
```

Comment:
put that atop the file, before the part that creates the hvac system please. (nitpicking, but this makes diffing easier because the sutff you care about is after the stuff you don't care about)